App/components/RenderLive/index.js

```diff
@@ -4,6 +4,32 @@ import style from './style';
 import LoadingSpinner from '../LoadingSpinner';
 
 const RenderLive = ({data, channelName, thumbImage, onPress}) => {
+  const renderLive = (item) => {
```

Comment:
renderEpisodes component have many similarity with this comp. can you merge both into one

Reply:
done